lib/services/authority.go

```diff
@@ -3,14 +3,40 @@ package services
 import (
 	"encoding/json"
 	"fmt"
+	"time"
 
+	"github.com/gravitational/teleport"
 	"github.com/gravitational/teleport/lib/defaults"
 	"github.com/gravitational/teleport/lib/utils"
 
 	"github.com/gravitational/trace"
 	"golang.org/x/crypto/ssh"
 )
 
+// CertParams defines all parameters needed to generate a host certificate.
+type CertParams struct {
+	PrivateCASigningKey []byte
```

Comment:
public members should have comments, also we are following these guidelines:

https://github.com/golang/go/wiki/CodeReviewComments